portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,77 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body{
+  background-color: #f4f4f4;
+  color: #555;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 16x;
+  line-height: 1.6em;
+  margin: 0;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.container{
+  width: 80%;
+    margin: auto;
+    overflow: hidden;
```

Comment:
The indentation seems different compared to elsewhere in file.

Reply:
fixing it